CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Incorrectly moving subfolder hierarchy from a deleted folder to a new folder at the same path during OneDrive incremental backup.
 - Handle calendar events with no body.
 - Items not being deleted if they were created and deleted during item enumeration of a OneDrive backup.
+- Skip huge OneNote items that Graph API prevents us from downloading
```

Comment:
nit: should put the size of the OneNote items here and/or link to a place that has the issue better documented. That way users of Corso have a way to see if something is wrong or if it's expected

Reply:
Good call, added.